saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/carconfig/DefaultHVACSettings.java

```diff
@@ -0,0 +1,22 @@
+package net.heberling.ismart.mqtt.carconfig;
+
+public class DefaultHVACSettings implements HVACSettings {
+    @Override
+    public Integer getMinAllowedTemp() {
+        return 16;
+    }
+
+    @Override
+    public Integer getMaxAllowedTemp() {
+        return 28;
+    }
+
+    @Override
```

Comment:
Maybe move this as default method into the interface?

Reply:
So, I am a bit busy this week... I just wanted to have a working state to start testing, and it is doing it´s job. 

The issue that is bothering me more right now is that the code between VehicleHandler and VehicleState is starting to grow fast and get out of control... :) Will try to do a first improvement there, but still thinking about how to properly manage the class structure to keep everything small.

With the settings, I am not sure yet if default methods are the way to go, or just a "default implementation". This will mainly depend on wether the class needs special functionality or not. I will try to add the MG4 relevant mapping in the coming days, if you know someone who can build the code from the branch and has a MG4 it would be great to let him /her test somewhen next week. The India/Australia variants seem a bit more hard to find people...